pkg/sandbox/sandbox.go

```diff
@@ -239,14 +220,21 @@ func (s *SandboxController) RegisterSandboxUser(userName string) error {
 			return false, err
 		}
 
-		klog.Info("Waiting...\n", userSignup)
 		for _, condition := range userSignup.Status.Conditions {
 			if condition.Type == toolchainApi.UserSignupComplete && condition.Status == corev1.ConditionTrue {
```

Comment:
Can u check in the if if the compliant status is set?

Reply:
addressed in https://github.com/redhat-appstudio/e2e-tests/pull/551/commits/c6b493edd06a981c0907b00b8a4462cc58245a00